src/main/java/core/mate/academy/service/BulldozerProducer.java

```diff
@@ -0,0 +1,15 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Bulldozer;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    @Override
+    public List<Bulldozer> get() {
+        Bulldozer bulldozer1 = new Bulldozer();
+        Bulldozer bulldozer2 = new Bulldozer();
+        Bulldozer bulldozer3 = new Bulldozer();
+        List<Bulldozer> bulldozers = List.of(bulldozer1,bulldozer2,bulldozer3);

```

Comment:
You could also try to initialize those machines without the need of creating every single field

Reply:
+

Reply:
`
List<Bulldozer> bulldozers = List.of(new Bulldozer(10), new Bulldozer(20), new Bulldozer(30));
`